api/internal/core/store/storehub.go

```diff
@@ -57,6 +57,19 @@ func InitStore(key HubKey, opt GenericStoreOption) error {
 		HubKeyGlobalRule:  true,
 		HubKeyStreamRoute: true,
 	}
+
+	hubsNeedCheckByCustomize := map[HubKey]bool{
+		HubKeySystemConfig: true,
+	}
+
+	if _, ok := hubsNeedCheckByCustomize[key]; ok {
+		validator, err := NewJsonSchemaValidator("customize." + string(key))
```

Comment:
Why not put this under "main"?

Reply:
`schema.json` is output from  `apisix`, my idea is to use different names for distinguishing them

Reply:
done